### PR TITLE
Amend usages of various (sub)commands

### PR DESCRIPTION
**Pull request by adlternative**
According to requirements, I changed the first character of the description of these subcommands to lowercase letters,
and removed the dots that may appear at the end of the sentence,
This is the first time I want to participate in the open source community,thanks.

Signed-off-by: adl <adlternative@gmail.com>

Thanks for taking the time to contribute to Git! Please be advised that the
Git community does not use github.com for their contributions. Instead, we use
a mailing list (git@vger.kernel.org) for code submissions, code reviews, and
bug reports. Nevertheless, you can use GitGitGadget (https://gitgitgadget.github.io/)
to conveniently send your Pull Requests commits to our mailing list.

Please read the "guidelines for contributing" linked above!

### CODE REVIEW

File: builtin/am.c [hunk 1]
```diff
@@ -2284,10 +2284,10 @@ int cmd_am(int argc, const char **argv, const char *prefix)
 			N_("skip the current patch"),
 			RESUME_SKIP),
 		OPT_CMDMODE(0, "abort", &resume.mode,
-			N_("restore the original branch and abort the patching operation."),
+			N_("restore the original branch and abort the patching operation"),
```
Comment (derrickstolee) [inline builtin/am.c:2287]: @adlternative: thanks for doing this cleanup.

You can help your commit message by following the standard pattern. I recommend something like:

```
builtin/*: update usage format

According to the "Error Messages" section of
Documentation/CodingGuidelines, we should not end in a full
stop or start with a capital letter. Fix old error and usage
messages to match this expectation.

(sign-off)
```

Feel free to use any or all of this in your commit message, but be sure the line width makes sense as I didn't check it.

Comment (adlternative) [reply builtin/am.c:2287]: ok,I will change it!By the way, I discovered that some of the checks failed, I will check again.

**Comment by dscho** [comment]
Please use your real name in the `Signed-off-by:` line, as required by the Git project.

**Comment by adlternative** [comment]
> Please use your real name in the `Signed-off-by:` line, as required by the Git project.

thanks for your reminder, i have set up a new PR in #822 ,I will standardize my format !

**Comment by dscho** [comment]
> > Please use your real name in the `Signed-off-by:` line, as required by the Git project.
> 
> thanks for your reminder, i have set up a new PR in #822 ,I will standardize my format !

Why a new PR? Why not just force-push and fix this PR?

**Comment by adlternative** [comment]
> > > Please use your real name in the `Signed-off-by:` line, as required by the Git project.
> > 
> > 
> > thanks for your reminder, i have set up a new PR in #822 ,I will standardize my format !
> 
> Why a new PR? Why not just force-push and fix this PR?

sorry! I know i shouldn't do this.I just get a big error in my local workarea(beca),which I reset or revent can't fix out ,i delete it Annoyingly ....And fork this repo again ...the bug disappared ....too strange ....
(Because I seemed to use a script to modify most of the entire project the day before yesterday, and later I modified it back, but my local test can't pass anyway, even if I use the fallback)

According to #822 , I will use `commit amend` to change sign-off

**Pull request closed without merging**